docs/agent-config.md

```diff
@@ -44,14 +59,6 @@ The following property may be used to disable it:
 |-------------------------------------------------------|-------------------------------------------------------|---------------------------------------------------------------------|
 | `otel.instrumentation.common.db-statement-sanitizer.enabled` | `OTEL_INSTRUMENTATION_COMMON_DB_STATEMENT_SANITIZER_ENABLED` | Enables the DB statement sanitization. The default value is `true`. |
 
-## Customizing the OpenTelemetry SDK
-
-*Customizing the SDK is highly advanced behavior and is still in the prototyping phase. It may change drastically or be removed completely. Use
-with caution*
-
-The OpenTelemetry SDK exposes SPI [hooks](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/spi)
-for customizing its behavior, such as the `Resource` attached to spans or the `Sampler`.
-
 Because the automatic instrumentation runs in a different classpath than the instrumented application, it is not possible for customization in the application to take advantage of this customization. In order to provide such customization, you can provide the path to a JAR file, including an SPI implementation using the system property `otel.initializer.jar`. Note that this JAR needs to shade the OpenTelemetry API in the same way as the agent does. The simplest way to do this is to use the same shading configuration as the agent from [here](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/cfade733b899a2f02cfec7033c6a1efd7c54fd8b/java-agent/java-agent.gradle#L39). In addition, you must specify the `io.opentelemetry.javaagent.shaded.io.opentelemetry.api.trace.spi.TraceProvider` to the name of the class that implements the SPI.
```

Comment:
I think this section is still relevant (though maybe too advanced for main agent config page, and scheduled to change a bit in #2442)

Reply:
yeah, I missed this line. I think we should remove it from this page, since it's not relevant for normal end-users and will probably confuse most people who are the target audience.